lib/sles4sap.pm

```diff
@@ -1452,6 +1454,28 @@ sub modify_selinux_setenforce {
     }
 }
 
+=head2 get_test_repos
+
+    Constructs an array of repos to be added to SUTs.
+
+=cut
+
+sub get_test_repos {
+    # In Incidents there is INCIDENT_REPO instead of MAINT_TEST_REPO
+    # Those two variables contain list of repositories separated by comma
+    set_var('MAINT_TEST_REPO', get_var('INCIDENT_REPO')) if get_var('INCIDENT_REPO');
+    my @repos = split(/,/, get_var('MAINT_TEST_REPO'));
+    # Add aggregate repos to @repos, if they are provided
+    # Test repos are expected to end in '_TEST_REPOS'
+    my @test_repos = grep { /_TEST_REPOS$/ } keys %bmwqemu::vars;
```

Comment:
Please add a comment in the code explaining about `TEST_REPOS`: https://github.com/openSUSE/qem-bot/blob/ecb7acc8badccce85969e05f368455390b1ab6eb/openqabot/types/aggregate.py#L104

I'd like to put emphasis on the fact that this particular part of OSADO code "is talking" to a particular qem-bot code.